File: cmd/numi/repl.py
def repl():
    while True:

        k = input("> ")
        
        if input == "exit":
            exit(0)

        print(k)

__all__ = [
    "repl"
]

if __name__ == "__main__":
    import os
    raise Exception(f"`{os.path.basename(__file__)} can\'t be ran as `{__name__}`")

File: cmd/numi/repl.js
const process = require("process")

function repl() {
    console.log("Not implemented")
    process.exit(0)
}

module.exports = {
    repl
}

if (require.main === module) {
    const path = require("path")

    console.error(`ERROR: \`${path.parse(__filename).name}\` file can\'t be ran as main.`)
    process.exit(1)
}